apps/docs-app/app/components/theme-page.gts

```diff
@@ -0,0 +1,257 @@
+import { array } from '@ember/helper';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import Button from '@nrg-ui/core/components/button';
+import Footer from '@nrg-ui/core/components/footer';
+import Form from '@nrg-ui/core/components/form';
+import Icon from '@nrg-ui/core/components/icon';
+import CardContainer from '@nrg-ui/core/components/mktg/card-container';
+import Header from '@nrg-ui/core/components/mktg/header';
+import SectionHeader from '@nrg-ui/core/components/mktg/section-header';
+import bind from '@nrg-ui/core/helpers/bind';
```

Comment:
All of these can be combined into a single import
```suggestion
import {
  bind,
  Button,
  Footer,
  Form,
  Icon,
  MktgCardContainer,
  MktgHeader,
  MktgSectionHeader
} from '@nrg-ui/core';
```

Reply:
I didn't comment on this before, but your `<template>` will need to be updated. `CardContainer` -> `MktgCardContainer`, `Header` -> `MktgHeader`, etc. This also needs a trailing comma (linting).